docs/reference/api/docker_remote_api_v1.22.md

```diff
@@ -3045,6 +3045,7 @@ Content-Type: application/json
   "Id": "7d86d31b1478e7cca9ebed7e73aa0fdeec46c5ca29497431d3007d2d9e15ed99",
   "Scope": "local",
   "Driver": "bridge",
+  "EnableIPv6": false,
```

Comment:
I think this is a spurious change, and this flag was not ther ein 1.22 API version